components/org.wso2.carbon.status.dashboard.web/src/auth/utils/AuthManager.jsx

```diff
@@ -0,0 +1,152 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+import AuthenticationAPI from '../../utils/apis/AuthenticationAPI';
+
+/**
+ * Name of the session cookie.
+ */
+const sessionUser = 'wso2statusdashboard_user';
```

Comment:
if this is the session cookie, then lets change the variable name to `SESSION_COOKIE_NAME`